crates/core/Cargo.toml

```diff
@@ -5,3 +5,12 @@ edition = "2021"
 description = "Extensible interactive autodiff."
 license = "MIT"
 include = ["src/lib.rs"]
+
+[dependencies]
+ts-rs = "6.1"
+serde = { version = "1", features = ["derive"] }
+serde-wasm-bindgen = "0.4"
```

Comment:
Can you make these Serde dependencies [optional](https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies) based on the feature you've defined below?